src/vs/workbench/api/common/extHostExtensionService.ts

```diff
@@ -699,6 +700,21 @@ export abstract class AbstractExtHostExtensionService extends Disposable impleme
 		);
 	}
 
+	private async _handleRemoteResolverEagerExtensions(): Promise<void> {
+		return Promise.all(
+			this._myRegistry.getAllExtensionDescriptions().map((desc) => {
+				if (this.isActivated(desc.identifier) || !isResolverExtension(desc, this._initData.remote.authority)) {
+					return;
+				}
+
+				return (
+					this._activateById(desc.identifier, { startup: true, extensionId: desc.identifier, activationEvent: `onResolveRemoteAuthority:${this._initData.remote.authority}` })
+						.then(undefined, err => this._logService.error(err))
+				);
+			})
+		).then(() => { });
+	}
```

Comment:
Could we replace this whole block with `return this._activateByEvent(`onResolveRemoteAuthority:${this._initData.remote.authority}`, false);` ?